consensus/src/phase.rs

```diff
@@ -61,6 +61,15 @@ impl<T: Operations + 'static, D: Database + 'static> Phase<T, D> {
 
         let size = call_phase!(self, get_committee_size());
 
+        let exclusion = match ctx.step.to_step_name() {
+            StepName::Proposal => None,
+            _ => Some(ctx.provisioners.get_generator(
```

Comment:
Will this re-execute sortition algorithm two times more per an iteration? Do you think it will be better if we can cache the generator key from Proposal step instead of re-calculating it on each step?

Reply:
Good catch! Going to reuse the previously generated committee